astropy/table/table.py

```diff
@@ -1753,6 +1757,14 @@ def _make_index_row_display_table(self, index_row_name):
         else:
             return self
 
+    @deprecated(
+        "6.1",
+        message="""show_in_notebook() is deprecated as of 6.1 and to create
+        interactive tables it is recommended to use dedicated tools like:
+        - https://github.com/bloomberg/ipydatagrid
+        - https://docs.bokeh.org/en/latest/docs/user_guide/interaction/widgets.html#datatable
+        - https://dash.plotly.com/datatable""",
```

Comment:
Have you checked how this deprecation message actually looks like?

Reply:
Looks decent-ish to me? (ipython)
<img width="1125" alt="Screenshot 2024-02-19 at 8 35 06 PM" src="https://github.com/astropy/astropy/assets/5363860/850978bf-2e90-44e5-a6d4-16492995481d">

(Jupyter lab interface)
<img width="1183" alt="Screenshot 2024-02-19 at 8 36 53 PM" src="https://github.com/astropy/astropy/assets/5363860/523714f5-851a-4d72-9dbf-c71053d18522">

Reply:
It looks to me like an extra space would improve alignment.
```suggestion
        message="""show_in_notebook() is deprecated as of 6.1 and to create
         interactive tables it is recommended to use dedicated tools like:
         - https://github.com/bloomberg/ipydatagrid
         - https://docs.bokeh.org/en/latest/docs/user_guide/interaction/widgets.html#datatable
         - https://dash.plotly.com/datatable""",
```